packages/web/src/pages/chat-page/components/ComposePreviewInfo.tsx

```diff
@@ -49,7 +49,7 @@ export const ComposerTrackInfo = (props: ComposerTrackInfoProps) => {
   const { trackId } = props
   const image = useTrackCoverArt2(trackId, SquareSizes.SIZE_150_BY_150)
 
-  const { data: track } = useGetTrackById({ id: trackId })
+  const { data: track } = useGetTrackById({ id: trackId }, { force: true })
```

Comment:
just curious why force is needed?

Reply:
really stinks. it's a caching race condition- some places we fetch playlists still don't have a user attached :/ I think that @schottra 's work may help here lots.

Reply:
Ouch. The SDK migration itself may not help since we re-use the same type for responses from multiple endpoints. But there is a cleanup work stream to split those types out that _will_ help.